docs/gatsby-config.js

```diff
@@ -72,7 +72,6 @@ const themeOptions = {
               'analyze/deployment-status',
               'analyze/statefulset-status',
               'analyze/image-pull-secrets',
-              'analyze/ingress',
```

Comment:
Why is ingress deleted? Is it not an actual analyzer?

Reply:
It doesn't work at all https://github.com/replicatedhq/troubleshoot/issues/264. The [code](https://github.com/replicatedhq/troubleshoot/blob/master/pkg/analyze/ingress.go#L12) is looking at storage classes.

Reply:
Oh man, that's rough. I'm torn then on removing documentation vs, say either fixing or removing the code.